tests/foreman/ui/test_settings.py

```diff
@@ -532,3 +532,52 @@ def test_positive_entries_per_page(session):
             assert str(total_pages) == page_content["Pagination"]["pages"]
         finally:
             setting_cleanup(setting_name=property_name, setting_value=default_property_value.value)
+
+
+@run_in_one_thread
+@tier2
```

Comment:
should we parameterize this kind of test?
`@pytest.mark.parametrize('download_policies', **parametrized(["Immediate", "On Demand"]))`
But this could add over head to test like creating org, product, deleting product and restoring setting.
No idea if it's worth doing in this case, still wanted to put it here.
Devendra, if you choose to parameterize it, don't forget to add `:parametrized: yes` docstring token. :)

Reply:
@jameerpathan111 Thanks for your input,  For now, I shifted this test case from UI to API, As you know the settings API test module is not a pytest, I am working on the conversion process and there I will definitely include your suggestion.